modules/social_features/social_landing_page/templates/node--featured--sky.html.twig

```diff
@@ -1,17 +1,18 @@
 {% extends "node--featured.html.twig" %}
```

Comment:
I missed where this file was introduced, but I think we should keep `--sky` templates in the `socialblue` template because they're a specialcase of that theme and not part of this module.

Reply:
Fixed and removed from here.